### ISSUE TITLE
Maven central index missing versions since 2.25.0

### ISSUE THREAD

**Issue by hazendaz** [blutorange/closure-compiler-maven-plugin#66]
note sure what is going on but wanted to report it.  See https://search.maven.org/artifact/com.github.blutorange/closure-compiler-maven-plugin.  Its been going on since 2.25.0.

**Comment by blutorange** [comment]
Thanks for reporting! But what exactly do you mean by missing? I see all versions listed here

![image](https://github.com/blutorange/closure-compiler-maven-plugin/assets/2456413/72838e36-5265-4696-a4e4-a62b1b0ceb5a)

And when I click on "Browse" it shows a list of all files, including the JAR, e.g. https://repo1.maven.org/maven2/com/github/blutorange/closure-compiler-maven-plugin/2.29.0/

**Comment by hazendaz** [comment]
The legacy maven central is missing it.  Its rare so I don't understand why.  I don't use the newer one its too slow.

https://search.maven.org/search?q=closure-compiler-maven-plugin

![Screenshot 2024-03-15 000837](https://github.com/blutorange/closure-compiler-maven-plugin/assets/975267/ddb9a7f4-f70e-472a-8df4-fa3e35547927)

It works fine, its just that it doesn't show up there so those using legacy central and ignoring push to the new since its usage/performance is very suspect won't know this updated.  Since I've known for a while I know to look for this specific plugin here in github to see what is out there.  No clue what the issue is.

**Comment by blutorange** [comment]
I see, that's strange. When I go to https://search.maven.org/search?q=closure-compiler-maven-plugin, it just immediately redirected me to the new page. If I search online, I usually use `mvn versions:display-dependency-updates` or https://mvnrepository.com/artifact/com.github.blutorange/closure-compiler-maven-plugin, so I didn't notice this.

The only relevant change between 2.25.0 and 2.26.0 that I can see is that I updated the nexus staging plugin from 1.6.8 to 1.6.13. I could image they changed something so that it only uses the newer one. For that is weird since it should use the same data basis.

It does say the old web interface will be gone soon though

> We are also no longer investing in additional features or functionality on https://search.maven.org/ and are planning a retirement date for that web interface in _late 2023/early 2024._

**Comment by hazendaz** [comment]
I've seen this on one or two projects on the flip side (new).  So I know they are not using the same data.  How it happens I'm unclear as I've only seen it on maybe 3 total repos.  I currently hate the new L&F because I look up 100s of libraries a week as we have to use a special process to procure them and scan them.  So the new UI is rather useless for that.  No matter how much it bugs me, I go back.  I did use it today just to see if it improved and it has not.  Take tomcat for example.  The way the new UI works, it no longer shows all latest items first but rather all in order.  Well tomcat 11, 10, 9, 8, etc, takes a while to go backwards.  I do lean on the bots (renovate / dependabot) as well but I have to still look stuff up.  I think maybe a bug ticket to sonatype is worth it here.  If nothing more maybe they can clarify what is happening.

As to nexus plugin, I don't use it myself.  Last I recalled all that does is help auto close / auto release if one so choices.  I do know at some point long time users like myself will have some things changed on us for sure.  It seems anyone newer releasing for first time has to do it in a different way now.  I've been doing it for maybe 10 years or more now.  So I'm in the legacy camp.  By any chance where you signed up more through their new way?  I know you have been on github for a long time but first few years was light (like my first year or two).  So maybe releasing wasn't until much later.

Anyway back to really what is a problem at my day job.  Since we have to go through a rather nonsense procurement process, unless we are very aware something got released, we have to look it up.  I support a super pom there with some 100 or so plugins.  I went some time with yours not getting updated during monthly cycles.  That ended up concerning me as I add release date info in properties as an attribute and anything that dates too long I consider possibly dormant which causes me to go poke around.  Then I see in fact you have releases out and they do pull.  That is the strange part, they are missing in the index but pull so I'd think the entire nexus process even with that plugin released fine.

It would be curious if you dropped back to the older copy if that magically caused the others to show.  Typically because I do it manually, the only time it fails to show up in the index is when I failed to close/release.  In your case, its in the new index so even stranger but does indicate they get data feeds differently and probably have some bug.
